repositories/createPost.ts

```diff
@@ -0,0 +1,74 @@
+import { ApolloClient, gql } from "apollo-boost";
+import Post from "@@/models/Post";
+import PostInputType from "@@/models/PostInputType";
+import { toPost } from "@@/serializers/graphql/post";
+
+export default async function createPost(
+  postInputType: PostInputType,
+  {
+    apolloClient,
+    authenticationToken,
+  }: {
+    apolloClient: ApolloClient<any>;
+    authenticationToken: string | null;
```

Comment:
Creating post always requires users authenticated.

```suggestion
    authenticationToken: string;
```